packages/shared/scope-hoisting/src/generate.js

```diff
@@ -12,7 +12,7 @@ export function generate(
 ) {
   let {code} = babelGenerate(ast, {
     minified: options.minify,
```

Comment:
@mischnic what does this tell babel to do? Should it also just be left to terser?

Reply:
AFAICT, it only compresses whitespaces:

```js
var generate = require("@babel/generator").default
var parse = require("@babel/parser").parse

const AST = parse(`const ABCDEF = "hel" + "lo";`);
console.log(generate(AST, {minified: true}).code);
// const ABCDEF="hel"+"lo";
```